docs/bundles/plain.md

```diff
@@ -2,43 +2,147 @@
 
 ## Overview
 
-This document is meant to define the `plain` bundle format as a reference for those publishing `plain` bundles
-for use with RukPak. For more information on the concept of a bundle, click [here](https://github.com/operator-framework/rukpak#bundle).
+This document is meant to define the plain bundle format as a reference for those publishing plain bundles for use with
+RukPak. A bundle is a collection of Kubernetes resources that are packaged together for the purposes of installing onto
+a Kubernetes cluster. A bundle can be unpacked onto a running cluster, where controllers can then create the underlying
+content embedded in the bundle. The bundle can be used as the underlying `spec.source` for
+a [`Bundle`](https://github.com/operator-framework/rukpak#bundle) resource.
+
+A plain bundle is a static collection of arbitrary Kubernetes YAML manifests. A plain bundle is contained in a directory
+that can be packaged in a container image layer , a `git` repository or any other content source that the
+[plain bundle provisioner](../provisioners/plain.md) supports.
+
+Supported source types for a plain bundle currently include the following:
+
+* A directory in a [container image](../sources/image.md)
+* A directory in a [`git` repository](../sources/git.md)
+* A set of keys in a [`ConfigMap`](../sources/local.md)
+* An [upload](../sources/upload.md)
+* A `.tgz` file returned by a [http endpoint](../sources/http.md)
 
-A plain bundle is simply a collection of static, arbitrary, Kubernetes YAML manifests in a given directory.
 
 The currently implemented plain bundle format is the `plain+v0` format. The name of the bundle format, `plain+v0`
-combines the type of bundle (plain) with the current schema version (v0).
+combines the type of bundle (plain) with the current schema version (v0). The
+[plain bundle provisioner](../provisioners/plain.md) is able to source
+`plain+v0` bundles and install them onto a Kubernetes cluster.
 
-> Note: the plain+v0 bundle format is at schema version v0, which means it's an experimental format that is subject
+> Note: the `plain+v0` bundle format is at schema version v0, which means it's an experimental format that is subject
```

Comment:
Could we follow Kubernetes version scheme with alpha, beta, GA? Ideally in the name `plain+v0`-> `plain+v1alpha1`? If not, at least offer a mapping in the documentation. Kubernetes authors have defined clear rules in terms of API evolution, which set clear expectations to the users. It would be beneficial to offer the same to RukPak users.

Reply:
I think this is unrelated to this PR, but definitely a good point. This is something that I feel would be worth creating and issue/discussion and doing as a follow-up.

Reply:
https://github.com/operator-framework/rukpak/issues/652